src/tests/EdFi.DataManagementService.Tests.E2E/Features/General/DiscoveryAPI.feature

```diff
@@ -1,34 +1,33 @@
 Feature: The Discovery API provides information about the application version, supported data model(s), and URLs for additional metadata.
 
-        @ignore
         Scenario: 01 GET / returns the root Discovery API document
-            Given a GET to the root of the API
+             When a GET request is made to "/"
```

Comment:
I remember this, and the reason of the `Given a GET to the root of the AP`I was because we have it in that way in the `Information.feature` file
```
Feature: Information
    Retrieve information about the API

        Scenario: 01 Load information
            Given a get to the root of the API
             Then retrieves the information about the API
```

But I think is better to keep the `When a GET request is made to "/"`

Reply:
Also, I was being a little bit lazy that I didn't want to write a new step definition 😆